beta/project-griffon/set-up-project-griffon.md

```diff
@@ -82,6 +127,24 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 Once the extension has been registered, you may begin a Project Griffon session using the following API.
 
 {% tabs %}
+{% tab title="Android" %}
+### startSession
+
+This API accepts a deep link in order to begin a session. When this API is called, the SDK will display a pin authentication overlay on your app to begin a session.
```

Comment:
- please remove "in order"

- Please reword the 2nd sentence so that it reads as follows:

"After this API is called, to begin a session, the SDK displays a PIN authentication overlay on your app ."

Reply:
Done.